core/src/main/java/io/github/tcdl/msb/monitor/aggregator/DefaultChannelMonitorAggregator.java

```diff
@@ -71,31 +74,52 @@ void onHeartbeatResponses(List<Message> heartbeatResponses) {
         for (Message msg : heartbeatResponses) {
             aggregateInfo(aggregatorStats, msg);
```

Comment:
cool, it would be nice to describe this "additional feature" about heartbeats in javadoc. Will add info in ChannelMonitorAggregator class Javadoc. Please take a look